src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,25 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {'Posted by '}
+        <UserInfo user={post.user} />
```

Comment:
user can be null, so it's better to add the check
```suggestion
        {post.user && <UserInfo user={post.user} />}
```
<img width="594" alt="image" src="https://github.com/user-attachments/assets/bdf8a0c3-708a-4fba-9ef6-deba65a8773e">